Description: Fix for cmake rules in azoth
 This patch is needed for successfull builds in Debian.
Author: Boris Pek <tehnick-8@yandex.ru>
Last-Update: 2011-10-10

--- a/src/plugins/azoth/CMakeLists.txt
+++ b/src/plugins/azoth/CMakeLists.txt
@@ -32,7 +32,7 @@ INCLUDE_DIRECTORIES (
 
 OPTION (ENABLE_QTMULTIMEDIA_KLUDGE "Enable kludges for finding QtMultimedia" OFF)
 
-IF ((${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} EQUAL 2.8 AND ${CMAKE_PATCH_VERSION} LESS 4) OR ENABLE_QTMULTIMEDIA_KLUDGE)
+#IF ((${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} EQUAL 2.8 AND ${CMAKE_PATCH_VERSION} LESS 4) OR ENABLE_QTMULTIMEDIA_KLUDGE)
 	IF (${QT_VERSION_MINOR} EQUAL 6)
 		INCLUDE_DIRECTORIES (/usr/include/qt4/QtMultimedia)
 		SET (QTMULTIMEDIA_KLUDGE "QtMultimedia")
@@ -41,7 +41,7 @@ IF ((${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} EQUAL 2.8 AND ${CMAKE_PATCH_V
 		INCLUDE_DIRECTORIES (/usr/include/QtMobility)
 		SET (QTMULTIMEDIA_KLUDGE "QtMultimediaKit")
 	ENDIF (${QT_VERSION_MINOR} EQUAL 6)
-ENDIF ((${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} EQUAL 2.8 AND ${CMAKE_PATCH_VERSION} LESS 4) OR ENABLE_QTMULTIMEDIA_KLUDGE)
+#ENDIF ((${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} EQUAL 2.8 AND ${CMAKE_PATCH_VERSION} LESS 4) OR ENABLE_QTMULTIMEDIA_KLUDGE)
 
 OPTION (ENABLE_CRYPT "Enable QCA2-based support for PGP" ON)
 
